assignments/assignment01.rb

```diff
@@ -52,14 +52,19 @@ def titleize(s)
 
 # Write your own implementation of `reverse` called `my_reverse`
 # You may *not* use the built-in `reverse` method
-def my_reverse(s)
-  # your implementation here
-end
 
-# Your method should generate the following results:
-my_reverse "Hello World"          #=> "dlroW olleH"
+def my_reverse(input)
 
-my_reverse "Goodbye Cruel World"  #=> "dlroW leurC eybdooG"
+  length = input.length
+  reverse = ""
+
+  for i in 1..length do
```

Comment:
While this type of for loop exists in ruby, it is completely non-idiomatic. There are many other ways to write loops in ruby.